cmd/prober/write.go

```diff
@@ -121,17 +122,33 @@ func rekorWriteEndpoint(ctx context.Context) error {
 	t := time.Now()
 	resp, err := http.DefaultClient.Do(req)
 	latency := time.Since(t).Milliseconds()
+	exportDataToPrometheus(resp, rekorURL, endpoint, POST, latency)
 	if err != nil {
-		fmt.Printf("error adding entry: %v\n", err.Error())
+		return fmt.Errorf("error adding entry: %w", err)
 	}
 
-	// Export data to prometheus
-	exportDataToPrometheus(resp, rekorURL, endpoint, POST, latency)
-
+	// If entry was added successfully, we should verify it
+	rekorClient, err := rclient.GetRekorClient(rekorURL, rclient.WithUserAgent(fmt.Sprintf("Sigstore_Scaffolding_Prober/%s", versionInfo.GitVersion)))
+	if err != nil {
+		return fmt.Errorf("creating rekor client: %w", err)
+	}
 	defer resp.Body.Close()
-	body, _ = io.ReadAll(resp.Body)
-	fmt.Println(string(body))
-	return nil
+	var logEntry models.LogEntry
+	err = json.NewDecoder(resp.Body).Decode(&logEntry)
+	if err != nil {
+		return fmt.Errorf("unmarshal: %w", err)
+	}
+	var logEntryAnon models.LogEntryAnon
+	for _, e := range logEntry {
+		logEntryAnon = e
+		break
+	}
```

Comment:
Curious if there's a more elegant way to extract the `models.LogEntryAnon`... please let me know!

Reply:
this is probably the best way 😅